### DISCUSSION TITLE
Translate nano.community to Portuguese-BR (pt-br)

### DISCUSSION THREAD

**Discussion by vinibarbosabr** [mistakia/nano-community#70] [Proposals]
With a goal to increasing the reach of nano.community's content, I propose to start translating the pages and directories into Portuguese-BR (pt-br), with the possibility of changing the language in the user's navigation.

In this way, we could also get input from a wider community that does not speak fluent English.

A counterpoint to this can be making it difficult to keep pages in different languages synchronized and updated. This will require a greater community effort.

Thoughts?

**Comment by mistakia** [comment]
It's a worthwhile effort. I'll start thinking about how to implement it and will follow up with some steps.

**Comment by mistakia** [comment]
localization/internationalization support is underway — https://github.com/mistakia/nano-community/pull/78
